README.md

```diff
@@ -17,9 +17,7 @@ NodeJs, Npm
 Github
 Visual Studio Code
 
-## Live Demo (if available)
-
-[Live Demo Link](https://livedemo.com)
+[Live Demo Link](https://shuhad786.github.io/My-portfolio-website/)
```

Comment:
#### OPTIONAL

- [ ] Kindly add back the deleted header of your live demo so users can easily find the live demo 👍🏿